app/scripts/lib/metrics.js

```diff
@@ -354,17 +351,17 @@ define([
      * @param {String} campaignId - marketing campaign id
      * @param {String} url - url of marketing link
      */
-    logMarketingImpression: function (campaignId, url) {
+    logMarketingImpression (campaignId, url) {
```

Comment:
Not sure if we can use [default argument values](https://babeljs.io/docs/learn-es2015/#default-rest-spread) here, something like:
```js
logMarketingImpression (campaignId = UNKNOWN_CAMPAIGN_ID, url) {
```